waitress/app/viewsets.py

```diff
@@ -91,7 +92,8 @@ def toggle_user_active_status(self, request, pk):
         user.save()
         return Response(serializer.data, status_code.HTTP_200_OK)
 
-    @action(methods=["get"], url_path="update-users", detail=False)
+    @csrf_exempt
```

Comment:
This is a security flaw, any action that updates the database must be protected with csrf, especially in a server side rendered app